lib/fortnox/api/models/edi_information.rb

```diff
@@ -0,0 +1,28 @@
+require "fortnox/api/models/base"
+
+module Fortnox
+  module API
+    module Model
+      class EDIInformation < Base
```

Comment:
I prefer fully qualified paths for these, `Fortnox ::API::Model::Base`. Partially because when you have several Base classes in your tree it's not always obvious, without mental lookup, to see which one is used. It's easy to assume things ;) The other reason has to do with refactoring. If I move this class into another module later it might inherit from another Base and you can get weird errors etc.